pkgs/top-level/all-packages.nix

```diff
@@ -16481,9 +16481,12 @@ with pkgs;
   openjdk = jdk;
   openjdk_headless = jdk_headless;
 
+  graalvmPackages =
+    recurseIntoAttrs (callPackage ../development/compilers/graalvm { });
   graalvmCEPackages =
     recurseIntoAttrs (callPackage ../development/compilers/graalvm/community-edition { });
   graalvm-ce = graalvmCEPackages.graalvm-ce;
```

Comment:
If we are going to create this `graalvmPackages`, I think we need to do a clean-up.

Please move all packages to be under the `graalvmPackages`, and make the `graalvmCEPackages` an alias to `graalvmPackages`.

Reply:
I thought that the others may not makes sense for now as they are not available for oracle?

Reply:
`graalvmCEPackages` was a mistake by itself because community edition is just a detail of the package, but since we didn't have a better reason to do so it was never fixed.

Since we are going to introduce `graalvmPackages` anyway, we can simply fix this mistake and deprecate `graalvmCEPackages` later.